kotlin-react-router-dom/src/main/kotlin/react/router/dom/Route.kt

```diff
@@ -3,9 +3,12 @@
 
 package react.router.dom
 
-import react.*
+import react.ComponentClass
+import react.ComponentType
+import react.Props
+import react.ReactElement
 
-external interface RouteProps : PropsWithChildren {
```

Comment:
Nooo... You was right 🙂

Reply:
Seems that in nearest future.
You can talk about it with Victor

Reply:
https://reactrouter.com/web/api/Route/route-render-methods
> The recommended method of rendering something with a <Route> is to use children elements